client/state/data-layer/wpcom/comments/index.js

```diff
@@ -120,7 +121,10 @@ export const announceFailure =
 			? translate( 'Could not retrieve comments for “%(postTitle)s”', { args: { postTitle } } )
 			: translate( 'Could not retrieve comments for post' );
 
-		dispatch( errorNotice( error, { duration: 5000 } ) );
+		const environment = config( 'env_id' );
```

Comment:
> . Looking at the code I think if comments fail to load for any reason it would either stay in the loading state or say "no comments" which is incorrect. 

That is how the error behaves on the full post page currently iirc, incorrectly saying "no comments" in that section. For inline comments we just dont render the inline comments section. Theres definitely some room for future improvement especially in the post-comment-list loading (or lack of loading) state.